### ISSUE TITLE
MoveToRegion problems.

### ISSUE THREAD

**Issue by dgokhan** [dmariogatto/Maui.Controls.BetterMaps#2]
It doesn't move when i use 'MoveToRegion' on the map. 
Anyone experiencing this problem? @dmariogatto 

![a.png](https://i.ibb.co/QbTMkKm/indir.png)

**Comment by dgokhan** [comment]
BugFix - MoveToRegion;

MapHandler.Android.cs -> Line 209
```
  private void MoveToRegion(MapSpan span, bool animate = true)
        {
            MessagingCenter.Subscribe<Map, MapSpan>(this, "MapMoveToRegion", (sender, arg) =>
            {

                if (PlatformView.GoogleMap is null) return;

                var ne = new LatLng(arg.Center.Latitude + arg.LatitudeDegrees / 2,
                    arg.Center.Longitude + arg.LongitudeDegrees / 2);
                var sw = new LatLng(arg.Center.Latitude - arg.LatitudeDegrees / 2,
                    arg.Center.Longitude - arg.LongitudeDegrees / 2);

                CameraUpdate update = CameraUpdateFactory.NewLatLngBounds(new LatLngBounds(sw, ne), 0);

                try
                {
                    if (animate)
                        PlatformView.GoogleMap.AnimateCamera(update);
                    else
                        PlatformView.GoogleMap.MoveCamera(update);
                }
                catch (IllegalStateException exc)
                {
                    System.Diagnostics.Debug.WriteLine($"MapHandler MoveToRegion exception: {exc}");
                }
            });
        }
```

**Comment by dmariogatto** [comment]
Thanks @dgokhan for the issue & investigation! I've fixed the `MessengerCenter` invocation in #4.
